orchagent/main.cpp

```diff
@@ -486,6 +486,9 @@ int main(int argc, char **argv)
     DBConnector config_db("CONFIG_DB", 0);
     DBConnector state_db("STATE_DB", 0);
 
+    /* Clears the ORCH_ABORT_STATUS flag in STATE_DB */
+    state_db.del(ORCH_ABRT);
```

Comment:
suggest have flush table at common place - https://github.com/sonic-net/sonic-buildimage/blob/master/files/scripts/swss.sh#L177

Reply:
That can be done, but clearing it here gives more buffer to the processes dependent on this flag.